21-sump-level-monitor/README.md

```diff
@@ -0,0 +1,103 @@
+# Sump Level Monitor
+
+Monitor water level changes in a sump using a Notecard and a float switch.
+
+- [Sump Level Monitor](#sump-level-monitor)
+  - [You Will Need](#you-will-need)
+  - [Notehub Setup](#notehub-setup)
+  - [Hardware Setup](#hardware-setup)
+  - [Notecard Firmware Setup](#notecard-firmware-setup)
+  - [Notecard Configuration](#notecard-configuration)
+    - [Notecard CLI](#notecard-cli)
+  - [Testing](#testing)
+  - [Going Further](#going-further)
+
+## You Will Need
+
+* USB A to micro USB cable
+* [Notecarrier-A](https://shop.blues.io/products/carr-al)
```

Comment:
Is there a specific feature on carr-a that we are using or could carr-b be used here instead?

Reply:
I think we could use either one. I'm not aware of the major differences aside from the antenna.